backend/app/services/video_service.py

```
@@ -185,6 +205,11 @@ def download_video(url: str, output_path: str) -> str:
         message = str(exc).lower()
         if "unsupported url" in message or "unsupported" in message:
             raise UnsupportedVideoFormatError("Unsupported URL or video format") from exc
+        if "sign in to confirm you're not a bot" in message or "confirm you’re not a bot" in message:
```

Ruff (0.15.5)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)